core/tests/protractor_desktop/emailDashboard.js

```diff
@@ -42,34 +41,6 @@ describe('Email Dashboard', function() {
     await waitFor.pageToFullyLoad();
   });
 
-  it('should query for users', async function() {
```

Comment:
Does this mean the dashboard is no longer usable?

Reply:
Yes, it is not usable until we add Beam jobs for it.

Reply:
Is this tracked explicitly somewhere under "things to do later (and soonish)"?

Reply:
Added here https://docs.google.com/spreadsheets/d/1Z1yrAO-SxjLs2FShg03PmN1obo27G-I29GiSf1lT6Rk/edit#gid=2068757808